examples/deployment/src/asa-helpers.js

```diff
@@ -0,0 +1,163 @@
+// Authors of example: Invertisment & amityadav0
+const algosdk = require('algosdk');
+
+// Function used to print created asset for account and assetid
+exports.printCreatedAsset = async function (deployer, account, assetid) {
```

Comment:
the function doesn't need deployer, it requires only `algodClient`. 
Let's update the parameter.

Reply:
I can't use algodClient directly. It doesn't have `waitForConfirmation` function.